ross/rotor_assembly.py

```diff
@@ -3821,10 +3821,21 @@ def rotor_example_6dof():
     >>> import ross as rs
     >>> import numpy as np
     >>> rotor6 = rs.rotor_assembly.rotor_example_6dof()
-    >>> camp6 = rotor6.run_campbell(np.linspace(0,400,101),frequencies=18)
 
-    # plotting Campbell Diagram
+    Plotting rotor model
+    >>> fig = rotor6.plot_rotor()
+    >>> fig.show()
```

Comment:
We should also remove `fig.show()` from the docstring examples, since the output representation of the figure is something that is hard to keep track of. I recommend just inserting a comment with '# to plot use fig.show()'.